app/main.py

```diff
@@ -1,6 +1,23 @@
+import os
+
+import requests
+
+
+API_KEY = os.environ.get("API_KEY")
+
+API_URL = f"https://api.weatherapi.com/v1/current.json?key={API_KEY}&q="
+
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print(f"Getting weather for {CITY}")
+    response = requests.get(f"{API_URL}?q={CITY}&key={API_KEY}")
```

Comment:
You are using a string formatting to construct the URL for the request. It's better to use the `params` parameter of the `requests.get()` function to pass the query parameters. This way, the requests library will take care of escaping the parameters correctly and you don't need to worry about it. Here is an example of how to do it:

python
params = {'q': CITY, 'key': API_KEY}
response = requests.get(API_URL, params=params)
```

This is a code style issue.